Cargo.lock

```diff
@@ -170,9 +170,9 @@ dependencies = [
 
```

Comment:
i think we should have potentially a separate runner or something that runs `cargo msrv verify` in one of the example projects ? (it seems to get confused by a workspace)

I'm currently using `cargo msrv list` to check this.